source/Game.py

```diff
@@ -1,14 +1,14 @@
+from __future__ import print_function
 from Tetramino import Tetramino
 from helper import print_matrix
-   
+
 class Game:
     def __init__(self):
         self.N_ROWS = 22
         self.N_COLUMNS = 10
 
         # PLAY_AREA: a list of strings where each string represents one row
-        self.PLAY_AREA = ['.' * self.N_COLUMNS for row in range(self.N_ROWS)]
-
+        self.PLAY_AREA = [['.' * self.N_COLUMNS] for row in range(self.N_ROWS)]
```

Comment:
Also, this does not exactly do what we wanted. It still represents rows as strings only. So, now, each row is represented like `['..........']` instead of the earlier `'..........'`. We need something like this:
```
self.PLAY_AREA = [list('.' * self.N_COLUMNS) for row in range(self.N_ROWS)]
```